### ISSUE TITLE
aursync: missing space in confirmation prompt

### ISSUE THREAD

**Issue by rmarquis** [AladW/aurutils#57]
Small implementation detail:

```
:: Proceed with transaction? [Y/n]y
```

You might want to add a space between the prompt and the given answer in `confirm()`.

**Comment by AladW** [comment]
It looks like this for me
```
* View build files? [Y/n] y
```
Due to the added space in:
```
printf -v message -- "$BOLD* $1 [Y/n]$ALL_OFF " "$@"
```

**Comment by rmarquis** [comment]
Do you mind updating the PKGBUILD in the aur too? Something is definitely wrong on my system, and I'm not sure which version is actually installed. Manual install with makpkg also failed the pkgver() update for some reason. I've never seen that issue before.

Another downside of using a custom repo for a helper is that it makes it cumbersome to run multiple helpers in parallel :/

**Comment by AladW** [comment]
Updated the AUR one (as you indicated, I forgot to `cd` in pkgver) vesion should be `r335.e9c99b8`

About running helpers in parallel, aursync uses AURDEST as of 0b6a43ee6ebe898392658246bbbc23d7190810f5, and PKGDEST can be set as usual. You do have to run repose manually after using a different helper. Feel free to open a new issue on that regard.

**Comment by rmarquis** [comment]
Doesn't happen anymore. I guess I was still using an old version - and the missing `cd` in the AUR  PKGBUILD prevented it to be correctly updated.

I was referring to the updating features of other helpers. Using a binary repo prevent them to see update in the AUR. I don't think there is a workaround, one simply needs to disable the [custom] repo in pacman.conf.

**Comment by AladW** [comment]
Not necessarily, see "Migrating foreign packages" in the updated README.

**Comment by rmarquis** [comment]
Great! I did miss the readme update. That will indeed help, thx.

**Comment by AladW** [comment]
I found out the reason behind the original issue: 69c4139aac59792f457643968d8eb6a54b15d4f3 introduced `pactrans` (essentially `pacman -Su`) to avoid building against old repo packages. #20

**Comment by AladW** [comment]
Fixed https://github.com/andrewgregory/pacutils/commit/63066ca4b9b352d737aee1d8b59d7dd2c55b6616
